src/python/gudhi/representations/metrics.py

```diff
@@ -227,14 +245,16 @@ class BottleneckDistance(BaseEstimator, TransformerMixin):
 
     :Requires: `CGAL <installation.html#cgal>`_ :math:`\geq` 4.11.0
     """
-    def __init__(self, epsilon=None):
+    def __init__(self, epsilon=None, n_jobs=None):
         """
         Constructor for the BottleneckDistance class.
 
         Parameters:
             epsilon (double): absolute (additive) error tolerated on the distance (default is the smallest positive float), see :func:`gudhi.bottleneck_distance`.
+            n_jobs (int): number of jobs to use for the computation. See :func:`pairwise_persistence_diagram_distances` for details.
```

Comment:
I put this comment here but it applies to all `See :func:`pairwise_persistence_diagram_distances` for details`.
I think it would be better to add a new joblib subsection in installation.rst / Run time dependencies section and link to that new subsection.

Reply:
I can add a section there, but here I am not trying to add a link to joblib in general. This is more to advertise the fact that I used `prefer="threads"` in this particular function, which is important for a user to understand how it interacts with their own use of parallelism. In other places in gudhi, we may use `prefer="processes"` (or no `prefer`), and we may know more about the functions called inside the parallel section.

Reply:
I added a paragraph about joblib in installation.rst.
(I meant to link to the section about scikit-learn below, but the link goes straight to the scikit-learn website, which I guess is fine)